language_names/additions.tsv

```diff
@@ -0,0 +1,149 @@
+en	lvs	Latvian
```

Comment:
Is it possible to use a TSV with headers? I'm pretty sure the Python lib supports it. This would be much more readable that way.

Reply:
Yeah it can be done